ui/account/src/main/java/app/tivi/account/AccountComponent.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.tivi.account
+
+import app.tivi.inject.ApplicationScope
+import com.slack.circuit.runtime.presenter.Presenter
+import com.slack.circuit.runtime.ui.Ui
+import me.tatarka.inject.annotations.IntoSet
+import me.tatarka.inject.annotations.Provides
+
+interface AccountComponent {
+    @IntoSet
+    @Provides
+    @ApplicationScope
+    fun bindAccountPresenterFactory(factory: AccountUiPresenterFactory): Presenter.Factory = factory
+
+    @IntoSet
+    @Provides
+    @ApplicationScope
+    fun bindAccountUiFactory(factory: AccountUiFactory): Ui.Factory = factory
```

Comment:
Do you think there's anything we could do to make these kind of stuff easier to use in other DI systems like kotlin-inject? We built anvil support mostly because that's just what we use, but open to supporting other stuff if there's stuff you think is reusable

Reply:
Another option is you could group these together into one composite factory, and possibly even cover multiple screens within it

Reply:
Personally I didn't have a problem with writing these. 

kotlin-inject makes creating the assisted injection factory super simple, which helped a lot.

I'm not sure how much of this stuff could be transferable across DI systems tbh.